### ISSUE TITLE
A11y_Azure data studio_Query History_Start/Pause Query History Capture_ContentOnHoverOver: When we hoverover with mouse to History, Pause, Maximize ,Close Panel button present on right navigation the tooltip is visible but it's not accessible with tab key or any other arrow keys.

### ISSUE THREAD

**Issue by Jtith** [microsoft/azuredatastudio#20876]
### GitHubTags: 
#A11yMAS; #A11yTCS; #A11ySev2; #SH_SQLAzureDataStudio_Win32_Oct2022; #SQL Azure Data Studio; #Win32; #FTP; #DesktopApp; #Keyboard; #NonBenchmark; #WCAG2.1.1; #Win11; 

### Environment Details:
Application Name: SQL Azure Data Studio
Windows 11 22H2 Version OS Build 22621.674
Screen Reader: NVDA 2020.4

### Repro Steps:

1. Install Azure Data Studio – Insiders
2. Tab till New connection and press enter.
3. Using valid credentials, connect to server.
4. Tab till extensions and press enter.
5. Tab till search box and search "Query History". 
6. Tab till Install and press enter.
7. Tab till new query and press enter
8. Tab till query editor and execute any query.
9. Open Toggle panel using Ctrl+J.
10. Tab till Query History tab and press enter.
11. Tab till "Start/Pause Query History Capture" button in the panel
12. You will observe that when we hover over with mouse tooltip is visible but with keyboard tooltip is not accessible with tab key or any other arrow keys.

### Actual:
When we hoverover with mouse to History, Pause, Maximize ,Close Panel button present on right navigation the tooltip is visible but it's not accessible with tab key or any other arrow keys.

### Expected:
When we navigate to History, Pause, Maximize, Close Panel button present on right navigation the tooltip should be accessible with tab key or any other arrow keys.

### User Impact:
Keyboard users will face difficulty in accessing the important tooltip information if not available and might not be able to recognize a particular control especially in absence of visual label.

### Attachment:
![20876_A11y_Azure data studio_Query History_Start_Pause Query History Capture_ContentOnHoverOver (1)](https://user-images.githubusercontent.com/100552367/196678949-7e5cd43b-0ea6-4f83-b440-7ddb861ce3e3.gif)

**Comment by Charles-Gagnon** [comment]
Dupe of https://github.com/microsoft/azuredatastudio/issues/19566 - so we should at least be able to downgrade this to sev3.

But in general @Jtith I'm confused at the requirement here. Is there actually an expectation that tooltips show up when you tab into a button? That'd get pretty annoying having that pop up constantly and is not something I've seen in other apps. Can you link to the WCAG item for this? (or other documented requirement)

**Comment by Jtith** [comment]
@Charles-Gagnon we can only close this bug. once the duplicate bug is fixed and closed.

**Comment by Jtith** [comment]
@Charles-Gagnon We are closing this bug as this issue is repro on throughout the application, we are tracking same on below bug and https://github.com/microsoft/azuredatastudio/issues/19566 bug would be closed once issue is fixed throughout the application.

**Comment by InduPriya1805** [comment]
GitHubTags:#SH-SQL Azure Data Studio-Win32-OCT2022

**Comment by Darshan962** [comment]
make sure all the elements a that display tooltips when hovered over with the mouse are focusable using the keyboard. Use aria-describedby to linkup the tooltips also make sure tooltips appear not only with mouse hover but also when keypad is pressed.
